src/Files.Backend/ViewModels/Shell/Multitasking/MultitaskingControlViewModel.cs

```diff
@@ -30,13 +33,57 @@ public TabItemViewModel SelectedItem
             set => SetProperty(ref _SelectedItem, value); // TODO(i): Wake up a sleeping tab
         }
 
+        public bool IsVerticalTabFlyoutEnabled
+        {
+            get => UserSettingsService.MultitaskingSettingsService.IsVerticalTabFlyoutEnabled;
+        }
+
         public IRelayCommand AddTabCommand { get; }
 
+        public IAsyncRelayCommand TabDroppedOutsideCommand { get; }
+
         public MultitaskingControlViewModel()
         {
             this.Tabs = new();
 
+            WeakReferenceMessenger.Default.Register<TabAddRequestedMessage>(this);
+
+            this.Tabs.CollectionChanged += Tabs_CollectionChanged;
+            this.UserSettingsService.OnSettingChangedEvent += UserSettingsService_OnSettingChangedEvent;
+
             AddTabCommand = new RelayCommand(() => AddTab());
+            TabDroppedOutsideCommand = new AsyncRelayCommand<TabItemViewModel>(TabDroppedOutside!);
+        }
+
+        private async Task TabDroppedOutside(TabItemViewModel tabItemViewModel)
+        {
+            if (!await ApplicationService.OpenInNewWindowAsync( /* TODO(i): path goes here */ ))
```

Comment:
@d2dyno1 If the new window tab is created only from a path argument, then I'd definitely be in favor of doing it this way without the serialization. 👍🏻